blockchain/chain_sync/src/sync.rs

```diff
@@ -947,7 +999,9 @@ mod tests {
             Cid::from_raw_cid("bafy2bzacecujyfvb74s7xxnlajidxpgcpk6abyatk62dlhgq6gcob3iixhgom")
                 .unwrap();
 
-        let root = cs.compute_msg_data(&[bls], &[secp]).unwrap();
+        let root =
+            ChainSyncer::<MemoryDB>::compute_msg_data(cs.chain_store.blockstore(), &[bls], &[secp])
```

Comment:
I'm a little lost on how this even compiles tbh
```suggestion
            ChainSyncer::compute_msg_data(cs.chain_store.blockstore(), &[bls], &[secp])
```